src/validate_total_storage_categorized.R

```diff
@@ -14,14 +14,13 @@ validate_total_storage_categorized <- function(data_categorized, n_hrus = 114958
   
   ##### Test: Only expected categories exist in the value column #####
   
-  # We know that one CA HRU doesn't have historic data for quantiles (104388) and will be Undefined
-  # This is also true for 7 other HRUs that are not in the U.S.
-  #problem_hruids <- c(104388, 46760, 46766, 46767, 82924, 82971, 82983, 82984)
-  #data_categorized_problem_hru <- data_categorized[data_categorized$nhru %in% problem_hruids,]
-  #data_categorized_good_hrus <- data_categorized[!data_categorized$nhru %in% problem_hruids,]
+  # In case we are missing historic data for quantiles their daily data will be Undefined
+  problem_hruids <- c(12099, 12107, 12108, 12692, 12731, 40871, 40880, 40882, 40956, 40957, 40971, 40979, 40983, 43167, 43168, 43171, 43172, 77834, 89590, 104167, 104168, 104173, 107629, 107649)
```

Comment:
these all have 0 for each quantile in the all_quantiles file so they are coming up as Undefined in our daily processing. 
![image](https://user-images.githubusercontent.com/6920344/100262143-5ad83380-2f11-11eb-825e-74f12764099e.png)

Reply:
Bummer, a lot more with undefined than last time.